<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>restaurantmenu</class>
 <widget class="QWidget" name="restaurantmenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>629</width>
    <height>569</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>629</width>
    <height>569</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>629</width>
    <height>569</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>5</y>
     <width>501</width>
     <height>101</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LayoutDirection::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;B Farnaz&quot;;
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>اتن</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 85, 0);
font: 10pt &quot;B Farnaz&quot;;</string>
   </property>
   <property name="text">
    <string>سبد خرید</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>631</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(&quot;C:/Users/Taheri/OneDrive/دسکتاپ/foodSource/hg3 copy.jpg&quot;);
background-repeat: no-repeat;
background-position: center;
</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>110</y>
     <width>631</width>
     <height>461</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LayoutDirection::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(60, 60, 60);
font: 13pt &quot;B Farnaz&quot;;
color: rgb(255, 85, 0);</string>
   </property>
   <property name="currentIndex">
    <number>4</number>
   </property>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>          غذا          </string>
    </attribute>
    <widget class="QListWidget" name="listWidgetMain">
     <property name="geometry">
      <rect>
       <x>2</x>
       <y>0</y>
       <width>621</width>
       <height>421</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 126, 52);</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>150</y>
       <width>291</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 126, 52);
color: rgb(60, 60, 60);
font: 18pt &quot;B Farnaz&quot;;</string>
     </property>
     <property name="text">
      <string>  آیتمی برای نمایش وجود ندارد...</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>          دسر          </string>
    </attribute>
    <widget class="QListWidget" name="listWidgetDessert">
     <property name="geometry">
      <rect>
       <x>2</x>
       <y>0</y>
       <width>621</width>
       <height>421</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 126, 52);</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>150</y>
       <width>291</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 126, 52);
color: rgb(60, 60, 60);
font: 18pt &quot;B Farnaz&quot;;</string>
     </property>
     <property name="text">
      <string>  آیتمی برای نمایش وجود ندارد...</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_5">
    <attribute name="title">
     <string>          نوشیدنی          </string>
    </attribute>
    <widget class="QListWidget" name="listWidgetDrink">
     <property name="geometry">
      <rect>
       <x>2</x>
       <y>0</y>
       <width>621</width>
       <height>421</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 126, 52);</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>150</y>
       <width>291</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 126, 52);
color: rgb(60, 60, 60);
font: 18pt &quot;B Farnaz&quot;;</string>
     </property>
     <property name="text">
      <string>  آیتمی برای نمایش وجود ندارد...</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_6">
    <attribute name="title">
     <string>         پیش غذا          </string>
    </attribute>
    <widget class="QListWidget" name="listWidgetStarter">
     <property name="geometry">
      <rect>
       <x>2</x>
       <y>0</y>
       <width>621</width>
       <height>421</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 126, 52);</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>150</y>
       <width>291</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 126, 52);
color: rgb(60, 60, 60);
font: 18pt &quot;B Farnaz&quot;;</string>
     </property>
     <property name="text">
      <string>  آیتمی برای نمایش وجود ندارد...</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_7">
    <attribute name="title">
     <string>          سایر          </string>
    </attribute>
    <widget class="QListWidget" name="listWidgetOthers">
     <property name="geometry">
      <rect>
       <x>2</x>
       <y>0</y>
       <width>621</width>
       <height>421</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 126, 52);</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>150</y>
       <width>291</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 126, 52);
color: rgb(60, 60, 60);
font: 18pt &quot;B Farnaz&quot;;</string>
     </property>
     <property name="text">
      <string>  آیتمی برای نمایش وجود ندارد...</string>
     </property>
    </widget>
   </widget>
  </widget>
  <zorder>widget_2</zorder>
  <zorder>label</zorder>
  <zorder>pushButton</zorder>
  <zorder>tabWidget_2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
